cmd/cloudx/project/update_test.go

```diff
@@ -46,15 +46,15 @@ func TestUpdateProject(t *testing.T) {
 			subcommand:      "project",
 			pathSuccess:     "fixtures/update/json/config.json",
 			pathFailure:     "fixtures/update/fail/config.json",
-			failureContains: "minimum 1 items allowed",
```

Comment:
I don't understand how this ever passed.. Both the fixture used and the error message in Ory Network are 2+ years old, but the error message definitely should the updated one...